homeassistant/components/tuya/sensor.py

```diff
@@ -592,6 +592,17 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Thermostat
+    # https://developer.tuya.com/en/docs/iot/f?id=K9gf45ld5l0t9
+    "wk": (
+        TuyaSensorEntityDescription(
+            key=DPCode.VALVE,
```

Comment:
According to the API specification by Tuya (as linked above), this data point is `valve_state` and not `valve`.

Reply:
Shared actual API response. The API response does not include valve state.

Reply:
Seems like an issue with your device, what did Tuya say?

Reply:
I've no idea how to contact them. The device works just fine, or actually the whole batch of them.

As mentioned, I might have to switch to another integration at this point.

Reply:
No replu from Tuya support over mail.